src/styles/footer.css

```diff
@@ -0,0 +1,94 @@
+.footer {
+    color: var(--bg-card-color);
+    width: 100%;
+    font-family: 'Montserrat', sans-serif;
+}
+
+.primary-footer {
+    background-color: var(--light-theme-default-color);
+    font-size: var(--h1);
+    color: var(--bg-card-color);
+    box-shadow: var(--box-shadow);
+    place-items: center center;
+    place-content: center center;
+}
+
+.secondary-footer {
+    background-color: #111111;
```

Comment:
Since I'm using this color in only 1 place, it did not make sense for me to create a separate variable. In case I use it in more places in the future, I shall change it 👍🏽